app/controllers/mixins/actions/vm_actions/resize.rb

```diff
@@ -3,16 +3,26 @@ module Actions
     module VmActions
       module Resize
         def resizevms
-          assert_privileges("instance_resize")
-          recs = find_checked_items
+          assert_privileges(params[:pressed])
+          # if coming in to edit from miq_request list view
+          recs = checked_or_params
```

Comment:
dead code? `recs` is never used before it is overwritten

Reply:
Ah, ok, the difference is that `checked_or_params` will also return `params[:id]` when no checkboxes are found.

Just making sure this was the intended result.